client/index.js

```diff
@@ -7,6 +8,10 @@ class Client {
     this.target = target
     this.logger = logger
   }
+  validateURL (value) {
+    var urlregex = /^(https?|ftp):\/\/([a-zA-Z0-9.-]+(:[a-zA-Z0-9.&%$-]+)*@)*((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]?[0-9])){3}|([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.(com|edu|gov|int|mil|net|org|biz|arpa|info|name|pro|aero|coop|museum|[a-zA-Z]{2}))(:[0-9]+)*(\/($|[a-zA-Z0-9.,?'\\+&%$#=~_-]+))*$/
```

Comment:
Just a heads up, I worked on a similar issue earlier this way and @bkeepers suggested to not to rely on a regular expression; instead use a existing npm package to validate the url.

Reply:
Actually, it was the same issue but in Probot repository. It's now shifted here because it's originating from here only.

Reply:
Hey! Thanks for informing.
>Just a heads up, I worked on a similar issue earlier this way and @bkeepers suggested to not to rely on a regular expression; instead use a existing npm package to validate the url.

 I would like to get the review and then change it.

>Actually, it was the same issue but in Probot repository. It's now shifted here because it's originating from here only.

Yeah I know and saw it earlier but original problem was occurring in this repository.

Reply:
Yes, i understand. I just gave a heads up. :wink: 
PS: PR looks good to me.